fetch.bs

```diff
@@ -2035,9 +2028,9 @@ not a <a>fresh response</a> or a <a>stale-while-revalidate response</a>.
 
 <hr>
 
-<p>The <dfn export for=response id=concept-response-location-url>location URL</dfn> algorithm of
-given a <a for=/>response</a> <var>response</var> is the following steps. They return null, failure,
-or a <a for=/>URL</a>.
+<p>The <dfn export for=response id=concept-response-location-url>location URL</dfn> algorithm, given
+a <a for=/>response</a> <var>response</var> and null or an <a for=/>ASCII string</a>
+<var>requestFragment</var> is the following steps. They return null, failure, or a <a for=/>URL</a>.
```

Comment:
I'd suggest:

> The **location URL** of a response _response_, given null or an ASCII string _requestFragment_, is the value returned by the following steps. It will be either null, failure, or a URL.

Reply:
"They return" is language from Infra. Agreed that what's before the dot is awkward dot and needs improving.